auto-run-dockerized-nondex/entrypoint.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+set -e
+
+# Checking if the git repo url is set
+if [ -z "$REPO_URL" ]; then
+  echo "Error: REPO_URL is not set. It is a required argument."
+  exit 1
+fi
+
+echo "Cloning repository: $REPO_URL"
+git clone "$REPO_URL" repo
+
+cd repo
+
+echo "Printing java version for reference"
+java --version
+
+# Initialize the Maven install command
+MVN_INSTALL_CMD="mvn clean install"
+
+# Add the -pl option if MODULE is provided
+if [ -n "$MODULE" ]; then
+  MVN_INSTALL_CMD="$MVN_INSTALL_CMD -pl $MODULE"
+fi
+
+# Output and execute the command
+echo "Running 'mvn clean install' with command: $MVN_INSTALL_CMD"
```

Comment:
`mvn clean install` often runs tests (many of which may be slow-running) in certain projects. Consider adding `-DskipTests=true` to the default install command

Reply:
Okay. To the contrary, do we not want to ensure that all unit tests are working correctly before we run the module on nondex? I can make this a configurable option as well for now.

Reply:
Actually, on second thought, -DskipTests does make sense here. Nondex will automatically filter out tests that fail on clean runs when it complies its final list of flaky tests.